imports/plugins/custom/vendor-onboarding/client/tourSteps.js

```diff
@@ -0,0 +1,128 @@
+export const dashboardSteps = [
+  {
+    intro: `<h4>Welcome to GRC Marketplace</h4>
+        <hr>
+        <div>This tour will help you get your shop up and running quickly</div>`
+  },
+  {
+    element: ".toolbar-vertical",
+    intro: `<h4>Vendor Dashboard</h4>
+        <hr>
+        <div>The dashboard allows you to manage your store, keep up with orders, choose your preferred payment methods and lots more</div>`,
+    position: "left"
+  },
+  {
+    element: ".fa-sun-o",
+    intro: `<h4> Orders </h4>
+        <hr>
+        <div> The orders section shows all the products ordered from yor store.</div>`,
```

Comment:
Please change "yor" to "your" here.

Reply:
Thank you for your review. I have fixed both issues now